app/renderer/components/Inspector/Source.js

```diff
@@ -78,18 +80,20 @@ class Source extends Component {
 
     return <div id='sourceContainer' className={InspectorStyles['tree-container']} tabIndex="0">
       {/* Must switch to a new antd Tree component when there's changes to treeData  */}
-      {treeData ?
-        <Tree
-          defaultExpandAll={true}
-          onExpand={setExpandedPaths}
-          expandedKeys={expandedPaths}
-          onSelect={(selectedPaths) => this.handleSelectElement(selectedPaths[0])}
-          selectedKeys={[path]}
-          treeData={treeData} />
-        :
-        <Tree
-          treeData={[]} />
-      }
+      <Spin size='large' spinning={!!methodCallInProgress && !!mjpegScreenshotUrl}>
```

Comment:
are you sure `!!mjpegScreenshotUrl` is correct? you only want to show the spinner when `mjpegScreenshotUrl` is set?